docs/editors/sublime.md

```diff
@@ -3,26 +3,26 @@ id: sublime
 title: Sublime Text
 ---
 
-Metals has experimental support for Sublime Text 3 thanks to
-[tomv564/LSP](https://github.com/tomv564/LSP).
+Metals works with Sublime Text 3 thanks to
+[sublimelsp/LSP](https://github.com/sublimelsp/LSP) and [scalameta/metals-sublime](https://github.com/scalameta/metals-sublime) plugins.
 
 ![Sublime Text demo](https://i.imgur.com/vJKP0T3.gif)
 
 ```scala mdoc:requirements
 
 ```
 
-## Installing the plugin
+## Installing the plugins
 
-First, install the LSP plugin:
-`Command Palette (Cmd + Shift + P) > Install package > LSP`
+Install the following packages:
 
-```scala mdoc:bootstrap:metals-sublime sublime
+- [sublimelsp/LSP](https://github.com/sublimelsp/LSP): Language Server Protocol support for Sublime Text.  
+`Command Palette (Cmd + Shift + P) > Install package > LSP`
 
-```
+- [scalameta/metals-sublime](https://github.com/scalameta/metals-sublime): For automatic installation of metals and custom endpoints. 
```

Comment:
```suggestion
- [scalameta/metals-sublime](https://github.com/scalameta/metals-sublime): For automatic installation of metals and custom commands. 
```
I'm not 100% sure what endpoints mean in this context. Is commands clearer? What do you think?

Reply:
It means commands like build-import, doctor-run, you are right it might be clearer